src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/VisitDto.kt

```diff
@@ -60,9 +62,12 @@ data class VisitDto(
   @Schema(description = "The visit modified date and time", example = "2018-12-01T13:45:00", required = true)
   @field:NotBlank
   val modifiedTimestamp: LocalDateTime,
-
+  @Schema(description = "What was the last application method", required = true)
+  @field:NotNull
+  val lastApplicationMethod: ApplicationMethod,
```

Comment:
Do we need this @hutcheonb-moj? would rather not include it, if front dose not use it?